src/gsy_e/gsy_e_core/sim_results/endpoint_buffer.py

```diff
@@ -45,6 +45,9 @@
     "max": None
 }
 
+scm_simulation_raw_data_validator = get_schema_validator("scm_simulation_raw_data_v1")
+simulation_raw_data_validator = get_schema_validator("simulation_raw_data_v1")
```

Comment:
If [this discussion](https://github.com/gridsingularity/gsy-framework/pull/421#discussion_r1086319896) reveals that we keep this interface: How about moving these into the constructor of the respective class? This way the SCM validator is not initialised when we have a normal simulation.

Reply:
Regardless of the result of our other discussion, it's possible that I move this into the constructor of the EndpointBuffer class if you agree.

Reply:
Done.